frontend/src/pages/projects/screens/detail/WorkspacesList.tsx

```diff
@@ -4,14 +4,23 @@ import EmptyDetailsList from './EmptyDetailsList';
 import { ProjectSectionID } from './types';
 import DetailsSection from './DetailsSection';
 import { ProjectSectionTitles } from './const';
+import { ProjectDetailsContext } from '../../ProjectDetailsContext';
+import { useNavigate } from 'react-router-dom';
 
 const WorkspacesList: React.FC = () => {
+  const navigate = useNavigate();
+  const { currentProject } = React.useContext(ProjectDetailsContext);
+  const projectName = currentProject.metadata.name;
   return (
     <DetailsSection
       id={ProjectSectionID.WORKSPACE}
       title={ProjectSectionTitles[ProjectSectionID.WORKSPACE]}
       actions={[
-        <Button key={`action-${ProjectSectionID.WORKSPACE}`} variant="secondary">
+        <Button
+          key={`action-${ProjectSectionID.WORKSPACE}`}
+          variant="secondary"
+          onClick={() => navigate(`/projects/${projectName}/spawner`)}
```

Comment:
I tried to use the Link component before, it will generate an underscore under the text when you hover the button, I am not sure whether we want the underscore or not so I changed it to this onClick function

Reply:
Oh hmmm... Yeah I guess the tile already had that link style 🤔 Okay, maybe we log something to address this -- there may be custom styles needed. Slide this point to the side, we need to still support this, and PF may need to understand this is one of those times we need them to help out.